segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkStorageManagerConfig.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.chunklayer;
+
+import io.pravega.segmentstore.storage.SegmentRollingPolicy;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+
+/**
+ * Configuration for {@link ChunkStorageManager}.
+ */
+@AllArgsConstructor
+@Builder(toBuilder = true)
+public class ChunkStorageManagerConfig {
+    /**
+     * Default configuration for {@link ChunkStorageManager}.
+     */
+    public static final ChunkStorageManagerConfig DEFAULT_CONFIG = ChunkStorageManagerConfig.builder()
+            .minSizeLimitForNativeConcat(0)
```

Comment:
We have removed the term "Native Concat", please rename accordingly.

Reply:
Thanks good catch.

Reply:
fixed